### PR TITLE
Upgrade Microsoft.Azure.WebJobs.Extentions.Storage and Microsoft.Communication.Sms Packages

### PR DESCRIPTION
**Pull request by zedy-wj**
1.Upgrade this project with the latest version of the package Microsoft.Azure.WebJobs.Extentions.Storage.

2.Upgrade this project with the latest version of the package Microsoft.Communication.Sms.

- Removing the class Azure.Communication.PhoneNumber, and using type string to replace it.

 
@scottaddie, @AlexGhiondea, @kyle-patterson , @tg-msft, @amishra-dev for notification

### CODE REVIEW

File: src/NotifyFunction/NotifyFunction.csproj [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <PackageReference Include="Azure.Communication.Sms" Version="1.0.0-beta.2" />
```
Comment (JoshLove-msft) [inline src/NotifyFunction/NotifyFunction.csproj:10]: There is a GA version of this package - [Azure.Communication.Sms](https://www.nuget.org/packages/Azure.Communication.Sms/). Can we update to 1.0.1?

**Review by JoshLove-msft — APPROVED** [review]

File: src/NotifyFunction/NotifyFunction.csproj (hunk 1 above)
Comment (zedy-wj) [reply src/NotifyFunction/NotifyFunction.csproj:10]: Class `Azure.Communication.PhoneNumber` does not exist in GA version.

Comment (JoshLove-msft) [reply src/NotifyFunction/NotifyFunction.csproj:10]: Looks like we would just pass the phone number as a string - https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/communication/Azure.Communication.Sms#send-a-11-sms-message

**Review by JoshLove-msft — CHANGES_REQUESTED** [review]
Let's update to the GA version of the SMS package so that the sample will be more useful to customers.

**Comment by zedy-wj** [comment]
@JoshLove-msft - Update according to your comments, please re-review it, thanks!

**Review by JoshLove-msft — APPROVED** [review]
Looks great!

**Comment by v-xuto** [comment]
@jongio Please help merge this PR. Thanks.

**Comment by v-xuto** [comment]
@jongio Please help merge this PR. Thanks.

**Comment by v-xuto** [comment]
@isaacrlevin Do you have permission to merge this PR? If you have, help merge this PR. Thanks a lot.

**Comment by v-xuto** [comment]
@jemed Do you have permission to merge this PR? If you have, help merge this PR. Thanks a lot.

**Comment by v-xuto** [comment]
@jemed Do you have permission to merge this PR? If you have, help merge this PR. Thanks a lot.

**Comment by v-xuto** [comment]
@jemed Do you have permission to merge this PR? If you have, help merge this PR. Thanks a lot.

**Comment by v-xuto** [comment]
@jongio I have contacted Jemed on Teams, he does not have permission to merge this PR. 
So, who should we ask to help merge?

**Comment by jongio** [comment]
@bradygaster - Could you please review/merge?  Thanks!

**Comment by bradygaster** [comment]
done, sir!

**Pull request merged**